src/redux/books/books.js

```diff
@@ -0,0 +1,42 @@
+const ADD_BOOK = 'bookstore-react-app/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore-react-app/books/REMOVE_BOOK';
+
+const booksReducer = (state = [], action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [
+        ...state,
+        {
+          id: action.id,
+          title: action.title,
+          author: action.author,
+        },
+      ];
```

Comment:
- [ ] **[OPTIONAL]** If you changed the action creator as suggested above you would need to also change this, like so:

  ```
  return [
    ...state,
    action.payload,
  ];
  ```

Reply:
@TedLivist 
thanks got it 👍